README.md

```diff
@@ -9,12 +9,16 @@ Warning: This is pre-alpha software. There's no guarantee anything works.
 
 Kaspad is the reference full node Kaspa implementation written in Go (golang).
 
-This project is currently under active development and is in a pre-Alpha state. 
+This project is currently under active development and is in a pre-Alpha state.
 Some things still don't work and APIs are far from finalized. The code is provided for reference only.
 
+## What is kaspa
+
+Kaspa is an attempt at a proof-of-work cryptocurrency that operates in internet speed, with subsecond block times, based on a generalization of Nakamoto Consensus, the PHANTOM protocol.
```

Comment:
What does it mean to "operate in internet speed?"
Also, I think it's enough to say that it's doing something rather than attempting to do something.